dynamodb/src/test/scala/akka/stream/alpakka/dynamodb/ExampleSpec.scala

```diff
@@ -0,0 +1,45 @@
+package akka.stream.alpakka.dynamodb
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.dynamodb.impl.DynamoSettings
+import akka.stream.alpakka.dynamodb.scaladsl.DynamoClient
+import akka.testkit.TestKit
+import com.amazonaws.services.dynamodbv2.model._
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+
+class ExampleSpec extends WordSpecLike with Matchers with BeforeAndAfterAll {
+
+  val localDynamo = new LocalDynamo(8001)
```

Comment:
perhaps use a dynamic port, can be grabbed with `akka.testkit.SocketUtil.temporaryServerAddress`